recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2019.10.11" %}
+{% set version = "2019.10.17" %}
```

Comment:
FYI yo don't need to update the version number here, since we aren't updating the global pinnings, just starting the migration.

Reply:
Updated to remove the version bump, thanks! I'm not 100% clear what information the migrators operate on. This should still work, even though the global pinnings already have this version?

Reply:
Yes, I think so. At the very least they should get a rebuild PR (even if the actual migration yaml is moot).